tests/commands/test-sdk.js

```diff
@@ -9,7 +9,7 @@
 
 const SDK = require('../../lib/commands/sdk');
 const path = require('path');
-const fs = require('fs-extra');
+// const fs = require('fs-extra');
```

Comment:
Is this require unused now?

Reply:
I've remove the `fs.removeSync...` parts so it is not used anymore.